autoGuides/autoGuides.js

```diff
@@ -128,13 +128,19 @@ AmCharts.autoGuidesProcess = function( chart, axis, dataProvider, categoryField
     /**
      * Populate days
      */
-    if ( config.days.indexOf( date.getDay() ) !== -1 ) {
+    if ( config.days.indexOf( AmCharts.useUTC ? date.getUTCDay() : date.getDay() ) !== -1 ) {
```

Comment:
The `start` and `end` should be local variables, not global. Please insert a `var start, end;` here.